src/gha_runner/gh.py

```diff
@@ -198,10 +199,15 @@ def wait_for_runner(self, label: str, wait: int = 15):
             The label of the runner to wait for.
         wait : int
             The time in seconds to wait between checks. Defaults to 15 seconds.
-
+        timeout : int
+            The maximum time in seconds to wait for the runner to be online.
+            Defaults to 300 seconds.
```

Comment:
not consistent?

Reply:
Fixed in [fa9c585](https://github.com/omsf-eco-infra/gha-runner/pull/22/commits/fa9c585e77f51a7190eed27c179d6451de5716bc)